app/config/config.yml

```diff
@@ -39,7 +39,7 @@ csa_guzzle:
 
 framework:
     assets:
-        version_strategy: irozgar_gulp_rev_versions.asset.gulp_rev_version_strategy
+        json_manifest_path: '%kernel.project_dir%/build/rev-manifest.json'
```

Comment:
just pointing out that https://github.com/elifesciences/journal/blob/9eee7beb02769c38320219bd444774013b04552d/smoke_tests.sh depends on this

Reply:
(and its schema)

Reply:
Yep, the file itself hasn't changed.